backend/src/routes/projectRouter.ts

```diff
@@ -225,15 +225,29 @@ projects.get("/v1/projects", async (req, res, next) => {
 });
 
 // Returns the session user's projects
-projects.get("/v1/members", async (req: Request, res, next) => {
-  if (req.query["user"]) {
-    const userId = req.query["user"];
-    const userMembers = await projectController.findUserProjects(
-      userId.toString()
-    );
-    res.json(userMembers);
+projects.get(
+  "/v1/members",
+  (req, res, next) => {
+    if (req.query["user"]) {
+      next();
+    } else {
+      res.status(401).json({ errors: ["unauthenticated"] });
```

Comment:
nit: This is the wrong error to throw since we are not checking if the user is logged in here, but whether the API call has a required parameter, it's semantically more correct to return a `400 INVALID_PARAMS` error instead, where the response might look like `{"errors":["user_missing"]}`